src/pages/case-competition.jsx

```diff
@@ -1,5 +1,32 @@
 import React from 'react';
+import background from '../assets/images/city-scape-background.png';
 
-export default class IRCJournal extends React.Component {
-
+export default class CaseCompetition extends React.Component {
+  render() {
+    return (
+      <div className="container">
+        <div className="section">
+          <div className="section__header">
+            1st Annual IRC Case Competition
```

Comment:
this is a page that should support multiple case competitions, right? you need to genericize case comp into components that can take arbitrary .md data to form each event, like a blog/list of events